util/HISTORY.md

```diff
@@ -13,6 +13,8 @@ latest
 - Added `dx_const`, `dy_const`, and `deglat` to grid generation code for doubly-periodic grids
 - Added f32 support to halo exchange data transformation
 - Use one single logger, from logging.py
+- Removed hard-coded values of `ak` and `bk` arrays and added in the feature to read in `ak` and `bk` values
+  from an ASCII file to compute the `eta` and `eta_v` values.
```

Comment:
You state you are reading from ascii, but it appears you chose to use NetCDF as the accepted input file-type.